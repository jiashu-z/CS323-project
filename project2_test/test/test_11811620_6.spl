struct A
{
  int num;
};

struct A
{
};

int test_11811620_5()
{
  int a;
  int b[10];
  struct A c;
  struct A d[10];

  a = c.num;
  a = a.num;
  c = d.num;
  a = c.n;


}