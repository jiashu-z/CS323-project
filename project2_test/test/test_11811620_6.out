Error type 15 at Line 6: redefine struct: A
Error type 13 at Line 18: accessing with non-struct variable
Error type 5 at Line 18: unmatching type on both sides of assignment
Error type 13 at Line 19: accessing with non-struct variable
Error type 5 at Line 19: unmatching type on both sides of assignment
Error type 14 at Line 20: no such member: n
Error type 5 at Line 20: unmatching type on both sides of assignment
